libs/island-ui/core/src/lib/Toast/Toast.tsx

```diff
@@ -72,7 +74,7 @@ export const ToastContainer: React.FC<ToastProps> = ({
         pauseOnHover
         transition={Slide}
       />
-      <style jsx>{toastKeyframes}</style>
+      {useKeyframeStyles && <style jsx>{toastKeyframes}</style>}
```

Comment:
More nextjs stuff... sorry about that.

Reply:
No worries, hard to escape these things completely. This fix works fine